app/models/assignment.rb

```diff
@@ -16,7 +16,7 @@ def authorized_for_delete?
 
   def self.add_volunteer_to_region volunteer, region
     return false if volunteer.new_record?
-    if Assignment.where(:volunteer_id=>volunteer, :region_id=>region.id).count==0
+    if Assignment.where(volunteer_id: volunteer, region_id: region.id).count == 0
```

Comment:
I think my vote goes to `.count == 0` because it will use a SQL `COUNT()` instead of loading the objects into memory and checking the contents of the array.

Could be switched to use [`Numeric#zero?`](https://ruby-doc.org/core-1.9.3/Numeric.html#method-i-zero-3F), but I think it's just fine as-is.